src/components/TokenPairSelector/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { TokenPairSelectorList } from 'components/TokenPairSelectorList'
+
+import ArrowWhite from 'assets/img/arrow-white.svg'
+
+const TokenPairSelectorStyled = styled.div`
```

Comment:
Do we specifically want the styled component in a separate file? And what is the threshold for when we opt to do this vs. just keeping it in the FC file?

Reply:
My opinion is that we shouldn't while the file is small
it's nice to have a small markup, and small css in a small file
Probably @W3stside  would disagree with me

Reply:
when scaffolding or starting and it is small, keep them in it's fine. But once it grows, move them. From there start to think about whether or not a component can be extrapolated into a more generic form and inherited around the app.

Reply:
But in this specific case, I don't think we should move it to it's own styled file.
I do think we should create a reusable button, and a reusable select. But being so small, it's totally fine for me to be in the same file

Reply:
my point is mainly that now, at this early re-factor stage, we should be doing this actively, instead of retro-actively. Largely because it is easiest now.

Keeping styled in one folder is fine, yes. Though that is what we started off doing and then files got huge and the rest is history. So while it's not the end of the world in anyway to have too many styled comps in a main component file, it's just my 2 cents